### PR TITLE
EASY-1609: Add Privacy Statement Link to Footer

### PR DESCRIPTION
**Pull request by lindareijnhoudt**
Fixes EASY-1609

#### When applied it will
* Remove address lines
* Add telephone number as link
* Add Privacy Statement link
* Change all link-targets to the english version and https

### CODE REVIEW

File: src/main/typescript/components/Footer.tsx [hunk 1]
```diff
@@ -78,29 +81,29 @@ const Link = ({ children, title, href }: React.AnchorHTMLAttributes<HTMLAnchorEl
...
         </Link>
         <Link title="View the legal information"
-              href="http://www.dans.knaw.nl/nl/over/organisatie-beleid/juridische-informatie">
+              href="https://dans.knaw.nl/en/about/organisation-and-policy/legal-information">
             Legal information
         </Link>
-        <Link title="View the property right statement"
-              href="http://www.dans.knaw.nl/en/about/organisation-and-policy/legal-information/property-rights-statement">
+        <Link title="View the property rights statement"
+              href="https://www.dans.knaw.nl/en/about/organisation-and-policy/legal-information/property-rights-statement">
```
Comment (rvanheest) [inline src/main/typescript/components/Footer.tsx]: this one still says `www` in the link, while others don't have that no more...

File: src/main/typescript/components/Footer.tsx [hunk 2]
```diff
@@ -25,13 +25,16 @@ const nwo = require("../../resources/img/footer/logo_NWO.png")
...
-        <li>Anna van Saksenlaan 51</li>
-        <li>2593 HW Den Haag</li>
         <li>
+            <a href="tel:+31-703494450">+31 70 349 44 50</a>
+        </li>        <li>
             <a href="mailto:info@dans.knaw.nl">info@dans.knaw.nl</a>
         </li>
         <li>
-            <a title="View how to contact" href="http://dans.knaw.nl/en/contact" target="_blank">More ›››</a>
+            <a title="View how to contact" href="https://dans.knaw.nl/en/contact" target="_blank">More ›››</a>
```
Comment (rvanheest) [inline src/main/typescript/components/Footer.tsx]: Switch privacy statement and `More >>>`. Seems more obvious to me.

File: src/main/typescript/components/Footer.tsx [hunk 3]
```diff
@@ -25,13 +25,16 @@ const nwo = require("../../resources/img/footer/logo_NWO.png")
 
 const ContactColumn = () => (
     <ul className="contact">
-        <li>Anna van Saksenlaan 51</li>
```
Comment (rvanheest) [inline src/main/typescript/components/Footer.tsx]: why no more address?

Comment (lindareijnhoudt) [reply src/main/typescript/components/Footer.tsx]: to be similar to the dans website footer

File: src/main/typescript/components/Footer.tsx (hunk 2 above)
Comment (lindareijnhoudt) [reply src/main/typescript/components/Footer.tsx]: the more is about the contact info.
first, i put the privacy statement link in the third column, with the other legal links. 
however, on the dans website it is on the first column, and chris asked to put it there

File: src/main/typescript/components/Footer.tsx (hunk 3 above)
Comment (rvanheest) [reply src/main/typescript/components/Footer.tsx]: I know, you changed that one as well. So in general: why no more address in both footers.
(unacceptable answer: "_to keep both of them in sync_" 😜 )

File: src/main/typescript/components/Footer.tsx (hunk 2 above)
Comment (rvanheest) [reply src/main/typescript/components/Footer.tsx]: No, I meant to put the `More >>>` below the privacy statement.

File: src/main/typescript/components/Footer.tsx (hunk 3 above)
Comment (lindareijnhoudt) [reply src/main/typescript/components/Footer.tsx]: the dans website is not the webui, but dans.knaw.nl

so, i changed both the deposit-ui and the webui to be more in line with the dans website

**Pull request merged**